llvm/lib/IR/Instruction.cpp

```diff
@@ -553,16 +553,17 @@ void Instruction::dropUBImplyingAttrsAndUnknownMetadata(
 }
 
 void Instruction::dropUBImplyingAttrsAndMetadata(ArrayRef<unsigned> Keep) {
```

Comment:
I think the `Keep` is no longer needed now, but I guess it doesn't hurt to keep it...